common/djangoapps/student/management/tests/test_manage_user.py

```diff
@@ -117,6 +117,16 @@ def test_wrong_email(self):
         self.assertIn('email addresses do not match', str(exc_context.exception).lower())
         self.assertEqual([(TEST_USERNAME, TEST_EMAIL)], [(u.username, u.email) for u in User.objects.all()])
 
+    def test_same_email_varied_case(self):
+        """
+        Ensure that the operation is continues if the username matches an
```

Comment:
nit: remove `is`. should be `Ensure that the operation continues if the username matches an`